src/t2iapi/device/types.proto

```diff
@@ -50,6 +50,15 @@ enum MdsOperatingMode{
     MDS_OPERATING_MODE_MAINTENANCE = 3;
 }
 
+/*
+Represents a type of Descriptor.
+Currently only filtering for MdsDescriptors is required.
+Will be extended as necessity arises.
+ */
+enum DescriptorType{
```

Comment:
Since AbstractDescriptor/Type exists, it might be a good idea to rename this to DescriptorClass or something similar. To avoid confusion and to keep the option to provide services with AbstractDescriptor/Type in the future.

Reply:
Renamed it.